EPPlus/Drawing/Chart/ExcelChartErrorBar.cs

```diff
@@ -0,0 +1,316 @@
+                        SetXmlNodeString(VALTYPEPATH, "stdErr");
+                        break;
+                    default:
+                        SetXmlNodeString(VALTYPEPATH, "fixedVal");
+                        break;
+                }
+            }
+        }
+
+        /// <summary>
+        /// This element specifies whether an end cap is not drawn on the error bars.
+        /// </summary>
+        public bool NoEndCap
+        {
+            get
+            {
+                return GetXmlNodeBool(NOENDCAPVALUEPATH, true);
+            }
+
+            set
+            {
+                SetXmlNodeBool(NOENDCAPVALUEPATH, value, true);
+            }
+        }
+
+        /// <summary>
+        /// Address range used for Custom value type
+        /// </summary>
+        public string MinusAddress
+        {
+            get
+            {
+                return GetXmlNodeString(_minusErrorPath);
+            }
+
+            set
+            {
+                CreateNode(_minusErrorPath, true);
+                SetXmlNodeString(_minusErrorPath, ExcelCellBase.GetFullAddress(_chartSerie._chartSeries.Chart.WorkSheet.Name, value));
+
+                CleanupCacheAndLit(_minusErrorCachePath, _minusErrorLitPath);
+             }
+        }
+
+        /// <summary>
+        /// Address range used for Custom value type
+        /// </summary>
+        public string PlusAddress
+        {
+            get
+            {
+                return GetXmlNodeString(_plusErrorPath);
+            }
+
+            set
+            {
+                CreateNode(_plusErrorPath, true);
+                SetXmlNodeString(_plusErrorPath, ExcelCellBase.GetFullAddress(_chartSerie._chartSeries.Chart.WorkSheet.Name, value));
+
+                CleanupCacheAndLit(_plusErrorCachePath, _plusErrorLitPath);
+            }
+        }
+
+        /// <summary>
+        /// This element specifies a value which is used with the Error Bar Type to determine the length of the error bars.
+        /// </summary>
+        public double Value
+        {
+            get
+            {
+                double? value =  GetXmlNodeDoubleNull(_errorBarValuePath);
+
+                switch(ValueType)
+                {
+                    case eErrorBarValueType.CustomErrorBars:
+                        throw new Exception("Error bar value is not valid for Custom Error Bars, use PlusAddress and MinusAddress");
+                    case eErrorBarValueType.StandardError:
+                        throw new Exception("Error bar value is not valid for Standard Error");
+                    case eErrorBarValueType.FixedValue:
+                        value = 0.1;
+                        break;
+                    case eErrorBarValueType.Percentage:
+                        value = 5;
+                        break;
+                    case eErrorBarValueType.StandardDeviation:
+                        value = 1.0;
+                        break;
```

Comment:
Unless I'm missing something, any of these 3 assignments will overwrite whatever value is retrieved by `GetXmlNodeDoubleNull`.

Consider rewriting this setter like this or similar:

```
                double defaultValue;
                switch (ValueType)
                {
                    case eErrorBarValueType.CustomErrorBars:
                        throw new Exception("Error bar value is not valid for Custom Error Bars, use PlusAddress and MinusAddress");
                    case eErrorBarValueType.StandardError:
                        throw new Exception("Error bar value is not valid for Standard Error");
                    case eErrorBarValueType.FixedValue:
                        defaultValue = 0.1;
                        break;
                    case eErrorBarValueType.Percentage:
                        defaultValue = 5;
                        break;
                    case eErrorBarValueType.StandardDeviation:
                        defaultValue = 1.0;
                        break;
                }

                return GetXmlNodeDoubleNull(_errorBarValuePath) ?? defaultValue;
```